packages/tokens/src/tokens.ts

```diff
@@ -539,4 +540,8 @@ export const tokens: Record<keyof Theme, string> = {
   curveEasyEaseMax: 'var(--curveEasyEaseMax)',
   curveEasyEase: 'var(--curveEasyEase)',
   curveLinear: 'var(--curveLinear)',
+
+  // ZIndexes
+  // This is a special case where the tokens contain default values
+  ...zIndexVarsWithDefaults,
```

Comment:
Let's just inline it there and avoid an additional file `./global/zIndexes`, i.e.:


```suggestion
  /**
   * Elevation 0
   * Can be used for background elements, like surfaces
   */
  zIndexBackground: 'var(--zIndexBackground, 0)',
  
  // ....
```